### ISSUE TITLE
Zephyr mining

### ISSUE THREAD

**Issue by h09s** [UnamSanctam/UnamWebPanel#402]
Hello, iam trying to mine Zephyr on CPU and i can you tell me what settings are good on it and what algorithm they use ?

**Comment by UnamSanctam** [comment]
Zephyr uses the RandomX algorithm, just like Monero, and you should also use the same settings as you would do with Monero.

**Comment by h09s** [comment]
iam iam not mistaken its the rx/0?

**Comment by h09s** [comment]
and is this pool good or no ? de.zephyr.herominers.com:1123

**Comment by UnamSanctam** [comment]
Yes rx/0 is correct (it's the short name for RandomX), and yes HeroMiners is a good pool though they do ban I believe, you can mine Zephyr on 2miners if you want to do that instead (since we know they don't ban).

**Comment by h09s** [comment]
and what is the minimum payout on 2miners because it doesn't say

**Comment by h09s** [comment]
and what about hashvault

**Comment by UnamSanctam** [comment]
On 2miners the minimum payout on the Zephyr pool is 0.1 ZEPH, HashVault also bans if you have a lot of low hashrate miners (they say so on their main page).

**Comment by h09s** [comment]
what could this mean ? 
![image](https://github.com/user-attachments/assets/110cdbc5-a4e3-431f-b754-8e2b36b17137)

**Comment by UnamSanctam** [comment]
Is it a CPU miner? Could be incorrect connection settings for the pool.

**Comment by h09s** [comment]
> Is it a CPU miner? Could be incorrect connection settings for the pool.

it only happens on this one every other is online or some says offline even tho they are online

**Comment by UnamSanctam** [comment]
Could be some problem with the CPU, operating system or something else, hard to know without any other information. If some say offline but they are actually online then that means that they cannot contact your web panel.
